tests/casp/oci_keyboard.pm

```diff
@@ -0,0 +1,36 @@
+# SUSE's openQA tests
+#
+# Copyright © 2017 SUSE LLC
+#
+# Copying and distribution of this file, with or without modification,
+# are permitted in any medium without royalty provided the copyright
+# notice and this notice are preserved.  This file is offered as-is,
+# without any warranty.
+
+# Summary: Change keyboard layout
+# Maintainer: Martin Kravec <mkravec@suse.com>
+
+use strict;
+use warnings;
+use base "y2logsstep";
+use testapi;
+
+sub run() {
+    # Switch to UK
+    send_key 'alt-e';
+    send_key 'up';
+
+    # Check that UK layout is active
+    send_key 'alt-g';
```

Comment:
Point of this test is that no error appears when switching layout and that we really type with selected layout - assert_screen 'keyboard-layout-uk' is checking that.

I don't like the idea of checking "something" changed in general. It is not reliable way to tell that we are not typing too fast, because "loading" process is already considered as success.

I can add wait_screen_change because I agree something should change when pressing alt-g.
I consider it abuse because for this reason we should wrap it around every send_key.

If I did not convince you I will change tomorrow :)